File: frontend-vue/src/types/customer.ts
export interface Customer {
  id: number;
  name: string;
  email: string;
  organizationNumber: string;
  createdAt: string;
  isDeleted: boolean;
}

export interface CustomerExportResponse {
  url: string;
  rowCount: number;
  truncated: boolean;
}

File: backend/src/Models/Customer.cs
namespace PortCom.Demo.Models;

public class Customer
{
    public int Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string OrganizationNumber { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
    public bool IsDeleted { get; set; }
}
